phoenix-core/src/main/java/org/apache/phoenix/jdbc/PhoenixStatement.java

```diff
@@ -287,10 +298,14 @@ public String toString() {
     private int queryTimeoutMillis;
     // Caching per Statement
     protected final Calendar localCalendar = Calendar.getInstance();
+    private boolean validateLastDdlTimestamp;
 
     public PhoenixStatement(PhoenixConnection connection) {
         this.connection = connection;
         this.queryTimeoutMillis = getDefaultQueryTimeoutMillis();
+        this.validateLastDdlTimestamp = this.connection.getQueryServices().getProps()
```

Comment:
Do you want to follow `getDefaultQueryTimeoutMillis` way to initialize `validateLastDdlTimestamp`?  I think it keeps the constructor clean.